draft-ietf-jsonpath-base.md

```diff
@@ -1065,17 +1064,16 @@ Queries:
 | `$[1:5:2]` | `"b"` <br> `"d"` | `$[1]` <br> `$[3]` | Slice with step 2 |
 | `$[5:1:-2]` | `"f"` <br> `"d"` | `$[5]` <br> `$[3]` | Slice with negative step |
 | `$[::-1]` | `"g"` <br> `"f"` <br> `"e"` <br> `"d"` <br> `"c"` <br> `"b"` <br> `"a"` | `$[6]` <br> `$[5]` <br> `$[4]` <br> `$[3]` <br> `$[2]` <br> `$[1]` <br> `$[0]` | Slice in reverse order |
-{: title="Array slice selector examples"}
+{: title="Array slice //PICKER// examples"}
 
-### Filter Selector
+#### Filter //PICKER//
 
-#### Syntax
+##### Syntax
 {: unnumbered}
 
-The filter selector has the form `[?<expr>]`. It works via iterating over structured values, i.e. arrays and objects.
+The filter //PICKER// has the form `?<expr>`. It works via iterating over structured values, i.e. arrays and objects.
```

Comment:
No. A _selector_ works by iterating over structured values. A //PICKER// matches, or fails to match, each of the values in the iteration in turn. //PICKER// descriptions need to avoid saying that a //PICKER// can _select_ a value, as this confuses //PICKERS// and selectors.

Reply:
I disagree.

- `1` matches specifically the second item of an array
- `name` matches specifically the "name" property of an object
- `?<expr>` iterates over all of the items and matches on values that make `<expr>` return true

The _selector_ doesn't iterate over the child values.

> //PICKER// descriptions need to avoid saying that a //PICKER// can select a value

The above text doesn't say a that a *picker can select a value.